source/ed/connectors/fusio_parser.cpp

```diff
@@ -436,6 +436,7 @@ void StopTimeFusioHandler::init(Data& data) {
     }
     id_c = csv.get_pos_col("stop_time_id");
     headsign_c = csv.get_pos_col("stop_headsign");
```

Comment:
```suggestion
    stop_headsign_c = csv.get_pos_col("stop_headsign");
```
otherwise, it's easy to be confused between tip and stop headsign.